src/matching.rs

```diff
@@ -653,23 +642,10 @@ pub fn matches_simple_selector<E>(selector: &SimpleSelector,
                 attr_value.ends_with(value)
             })
         }
-
-        SimpleSelector::AnyLink => {
+        SimpleSelector::NonTSPseudoClass(ref pc) => {
             *shareable = false;
-            element.is_link()
-        }
-        SimpleSelector::Link => {
-            element.is_unvisited_link()
+            element.match_non_ts_pseudo_class(pc.clone())
```

Comment:
I missed this at the time but this should pass a reference instead of cloning. Let’s change that together with the next breaking change.